scripts/features/eventstore.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+# Check if EventStore has been installed
+if [ -f /home/vagrant/.homestead-features/eventstore ]
+then
+    echo "EventStore already installed."
+    exit 0
+fi
+
+touch /home/vagrant/.homestead-features/eventstore
+chown -Rf vagrant:vagrant /home/vagrant/.homestead-features
+
+# Determine wanted version from config
+set -- "$1"
+IFS=".";
+
+if [ -z "${version}" ]; then
+    installVersion="" # by not specifying we'll install latest
+else
+    installVersion="=${version}"
+fi
+
+# Install repository
+curl -s https://packagecloud.io/install/repositories/EventStore/EventStore-OSS/script.deb.sh | sudo bash
+
+# Install EventStore package
+sudo apt-get install -y eventstore-oss"$installVersion"
```

Comment:
We can remove `sudo` here since features will always be executed as `root`

Reply:
Of course :)